107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -126,6 +111,30 @@ public CacheManager getCacheManager(URI uri, ClassLoader classLoader, Properties
     return cacheManager;
   }
 
+  public Eh107CacheManager createCacheManager(Configuration config) {
+    return createCacheManager(config, null, new Properties());
+  }
+
+  public Eh107CacheManager createCacheManager(Configuration config, URI uri, Properties properties) {
```

Comment:
I would invert `URI` and `Configuration` parameters and make this one use the proper cache manager registration.